controllers/userController.js

```diff
@@ -0,0 +1,34 @@
+const userService = require("../services/userService");
+const { catchAsync } = require("../utils/error");
+
+const signIn = catchAsync(async (req, res) => {
+  const { kakaoAccessToken } = req.body;
+
+  const data = await userService.getKakaoUserInfo(kakaoAccessToken);
+
+  return res.status(200).json(data);
+});
+
+const userUpdate = catchAsync(async (req, res) => {
+  const userId = req.user.id;
+  const { first_name, last_name, birth, mobile_number, email } = req.body;
+
+  if (!userId) {
+    return res.status(400).json({ message: "Required UserToken" });
+  }
+
+  const result = await userService.userUpdate(
```

Comment:
함수 이름은 다음과 같이 행위를 앞쪽으로 배치해주는 것이 좋을 것 같아요
```suggestion
  const result = await userService.updateUserInfo(
```